velox/docs/functions/spark/string.rst

```diff
@@ -154,6 +154,14 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT startswith('js SQL', 'SQL'); -- false
         SELECT startswith('js SQL', null); -- NULL
 
+.. spark:function:: str_to_map(text[, pairDelim[, keyValueDelim]]) -> map(string, string)
+
+    Create a map after splitting the ``text`` into key/value pairs using delimiters.
```

Comment:
Can you explain the constraint for delimiter? Does `;;` supported?

Reply:
In the current impl., the constraint is only single character is accepted as delimiter. So `;` is allowed.